JuliaInterface/example/function_perform.jl

```diff
@@ -0,0 +1,25 @@
+module Stuff
+
+import Main.ForeignGAP: MPtr
+
+function myfunc( a::MPtr, b::MPtr )
+    return a;
+end
+
+function myfunc( a::MPtr, b::Int64 )
+    return a;
+end
+
+function myfunc( a::Int64, b::MPtr )
+    return a;
+end
+
+function myfunc( a::Int64, b::Int64 )
+    return a;
+end
+
+end
+
+function blafunc(a, b)
```

Comment:
I understand that this is just my hacked together example, but if this is added to the repo, perhaps let's choose some better names? E.g. 
* Stuff -> function_perform to match the file (and BTW, perhaps use `function_calls` instead? That'd at least match the GAP nomenclature, and also fits with the `jl_call*` function names
* myFunc -> returnFirstWithMethodDispatch
* blafunc -> returnFirstGeneric

Well, or something along these lines.